code/modules/client/preferences_savefile.dm

```diff
@@ -394,6 +394,13 @@ SAVEFILE UPDATING/VERSIONING - 'Simplified', or rather, more coder-friendly ~Car
 	READ_FILE(S["feature_moth_wings"], features["moth_wings"])
 	READ_FILE(S["feature_moth_markings"], features["moth_markings"])
 
+	//VOID EDIT BEGIN
+	READ_FILE(S["feature_ipc_screen"], features["ipc_screen"])
+	READ_FILE(S["feature_ipc_antenna"], features["ipc_antenna"])
+	READ_FILE(S["feature_ipc_chassis"], features["ipc_chassis"])
```

Comment:
There's actually another way to do this if you want it to be modular;
Basically since this is called by /world/New(), you can either add it there, or add our own /world/New() overwrite that would call parent
Example can be seen here for how its done:
https://github.com/fulpstation/fulpstation/blob/9dcd4208693fed46d16e4655e39bcc169e7640c9/fulp_modules/features/species/species_integration.dm#L15
https://github.com/fulpstation/fulpstation/blob/9dcd4208693fed46d16e4655e39bcc169e7640c9/fulp_modules/fulp_world.dm#L9